pages/manage/productions/create.vue

```diff
@@ -42,16 +42,14 @@
           />
         </section>
         <section v-else>
-          <b-form-group :label="$t('productions.create.production_name')">
-            <b-form-radio
-              v-for="production in availableProductions"
-              :key="production.id"
-              v-model="selectedSuggestedProductionId"
-              :value="production.id"
-              name="production"
-              >{{ production.title }}</b-form-radio
-            >
-          </b-form-group>
+          <pub-radio-group
+            v-model="selectedSuggestedProductionId"
+            :group-label="$t('productions.create.production_name')"
+            :values="availableProductionIds"
+            :labels="availableProductionTitles"
+            name="production"
+            class="choose-suggested-production"
+          />
```

Comment:
I like how you made it a single component that takes a list of values and labels and emits the selected value instead of forcing us to replicate this logic in every place we show radio buttons 👍